playwright-e2e/playwright.config.ts

```diff
@@ -16,15 +16,16 @@ const testConfig: PlaywrightTestConfig = {
   globalSetup: require.resolve('./fixtures/global-setup'),
   testDir: '.',
   testMatch: '**/*.spec.ts',
-  /* Maximum time one test can run for. Test should be short and takes less than 2 minutes to run */
-  timeout: 120 * 1000,
+  /* Maximum time one test can run for. Test should be short and takes less than 4 minutes to run */
+  timeout: 10 * 60 * 1000,
```

Comment:
These timeouts are way too long.  Hoping to come up with a better way to manage timeouts.